jwt-tutorial-005/src/main/java/org/jwt/server/Config.java

```diff
@@ -0,0 +1,12 @@
+package org.jwt.server;
+
+import org.springframework.stereotype.Component;
+
+import com.levigo.jadice.web.server.config.ConfigurationManager;
+
+@Component
+public class Config {
+    Config(){
```

Comment:
Vorschlag:

```
@Value("${webtoolkit.tileCachingEnabled:false}")
private boolean cachingEnabled;
```
Als Feld oben einfügen. Dann kann man unten in Zeile 10 das boolean aus der Konfiguration (application.yaml) übergeben.

Ansonsten müsste diese Klasse dokumentiert werden, warum immer false ist, auch wenn man ggf true in die application.yml schreibt :-)

Reply:
done, bitte mal nachgucken ob es so gemeint hat